.buildkite/build.rayci.yml

```diff
@@ -78,6 +78,7 @@ steps:
   - label: ":tapioca: build: doc"
     instance_type: medium
     commands:
+      - bazel build //:install_py_proto
```

Comment:
@aslonnie can you review the CI related changes. Now doc build requires the auto-generated python proto files.

Reply:
maybe add it in the makefile? or this will be one extra step that people work on docs need to remember to run.